migrations/Version20180419183337.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Migrations;
+
+use Doctrine\DBAL\Migrations\AbortMigrationException;
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+class Version20180419183337 extends AbstractMigration
+{
+  /**
+   * @param Schema $schema
+   * @throws AbortMigrationException
+   */
+  public function up(Schema $schema) {
+    // this up() migration is auto-generated, please modify it to your needs
+    $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+
+    $this->addSql('ALTER TABLE user_settings ADD use_gravatar TINYINT(1) NOT NULL COMMENT \'(DC2Type:boolean)\'');
+  }
+
+  /**
+   * Set newly added use gravatar flag to true.
+   * @param Schema $schema
+   */
+  public function postUp(Schema $schema) {
+    $this->connection->executeQuery("UPDATE user_settings SET use_gravatar = TRUE");
```

Comment:
Is TRUE actually allowed here?

Reply:
Well it worked on my local database (expectably) and even on our dev environment... So I would think so.

Reply:
It is far better, to use specific functions for specific purposes. Instead of `executeQuery`, you should have used `update()`. https://www.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/data-retrieval-and-manipulation.html#update